File: assignment 22/p1.c
#include<stdio.h>
int CountCapital(char *);
int main()
{
char str[20];
int iret=0;
printf("enter string \n");
scanf("%[^'\n']s",str);
iret=CountCapital(str);
printf("count of capitalletters in a string is\n %d",iret);
return 0;
}
int CountCapital(char *ptr)
{
int icnt=0;
while(*ptr!='\0')
{
if(*ptr>='A'&&*ptr<='Z')
{
icnt++;
}
*ptr++;
}
return icnt;
}

File: assignment 31/P1.java
import java.util.*;
class Mangesh
{
	int CountCapital(String str)
	{
		int icnt=0, i=0; 
		char arr[]=str.toCharArray();
		int ilength= str.length();
		for(i=0;i<ilength;i++)
		{
			if(arr[i]>='A'&&arr[i]<='Z')
			{
				icnt++;
			}

		}
		return icnt;
	}
	
}
class P1
{
	public static void main(String[] args) 
{			
	int iret=0;
Scanner sc = new Scanner(System.in);
System.out.println("enter String");
String s= sc.nextLine();
Mangesh mobj= new Mangesh();
iret=mobj.CountCapital(s);
System.out.println("no of capital lettres are"+iret);
}
}